tests/dummy/app/controllers/infinity.js

```diff
@@ -0,0 +1,18 @@
+import Ember from 'ember';
+const { get, set } = Ember;
+
+const images = ["jarjan", "aio___", "kushsolitary", "kolage", "idiot", "gt"];
+
+const models = [...Array(10).fill().map(() => `https://s3.amazonaws.com/uifaces/faces/twitter/${images[(Math.random() * images.length) | 0]}/128.jpg`)];
+
+export default Ember.Controller.extend({
+  models,
```

Comment:
This route uses optimistic rendering.  Guessing we don't want to control this correct and leave that up to the users?  i.e. 2 requests will be made - first for the 1st set of records.  While that is pending, it will trigger `infinityLoad()`.